source/administration/custom-terms-of-service.rst

```diff
@@ -0,0 +1,53 @@
+.. _custom-terms-of-service:
+
+Custom Terms of Service (Beta) (E20)
+=====================================
+
+*Available in `Enterprise Edition E20 and higher <https://about.mattermost.com/pricing/>`_.*
+
+In Mattermost Enterprise Edition E20, you can outline custom Terms of Service for your team members to accept before they can access Mattermost on web or desktop.
+
+.. note::
+
+ Users on mobile do not have to accept the terms of service with mobile support scheduled for an upcoming release.
```

Comment:
Consider revising "Users on mobile will not be presented with the custom Terms of Services.  Mobile support is scheduled for an upcoming release."  '...do not have to accept' may imply that the functionality is different, when it is not available yet.

Reply:
Good piece of feedback, updated